.github/pull-request-template.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Pull request template
+about: 'type(scope): Subject of the pull request '
+---
+
+***Issue***: Include link to the Jira/Github Issue
+
+***Description:***
+Provide a detailed description of the changes made by this pull request.
+
+***Additional Info:***
+Include any other relevant information such as how to use the new fuctionality, screenshots, etc.
+
```

Comment:
Unfortunately, this is not the latest template. 😭

Look at https://github.com/lacework/terraform-aws-cloudtrail/blob/main/.github/pull-request-template.md